### ISSUE TITLE
Docker failed to initalize Windows 10

### ISSUE THREAD

**Issue by JaskSkull** [docker/for-win#12233]
**Expected behavior**

Docker desktop should open

**Actual behavior**

System.NullReferenceException: Object reference not set to an instance of an object.

   at Docker.Core.Features.Features.<IsEnabled>d__16.MoveNext() in C:\workspaces\master-merge\src\github.com\docker\pinata\win\src\Docker.Core\Features\Features.cs:line 91

--- End of stack trace from previous location where exception was thrown ---

   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Docker.App.<Initialize>d__28.MoveNext() in C:\workspaces\master-merge\src\github.com\docker\pinata\win\src\Docker.Desktop\App.cs:line 114

--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Docker.Core.TaskExtension.<Forget>d__0.MoveNext() in C:\workspaces\master-merge\src\github.com\docker\pinata\win\src\Docker.Core\Extensions.cs:line 30

--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

**Information**

- Docker version: 4.1.0.69386
- Windows 10 Home
- Are you running inside a virtualized Windows e.g. on a cloud server or a mac VM: No

**Comment by amjados** [comment]
i install docker on virtualbox windows 10. 

any help ?

**Comment by amjados** [comment]
i have same error 

i install docker on virtualbox windows 10

**Comment by amjados** [comment]
i have same error

i install docker on virtualbox windows 10

any help !!!

**Comment by amjados** [comment]
i have same error

i install docker on windows 10 on virtualbox

any help !!!

**Comment by JaskSkull** [comment]
> i have same error
> 
> i install docker on windows 10 on virtualbox
> 
> any help !!!

I got it working. I had to uninstall docker then install it again and restart the PC. Every time it didn't' work I had to delete al docker related folders.

Afterwards, I had to install WS2 and Ubuntu inside WS2. I followed this [tutorial](https://www.youtube.com/watch?v=_et7H0EQ8fY&t=153s), it is in spanish but if you don't understand it just mute him and follow what he is doing.

I hope it helps.
